src/main/java/io/github/thebusybiscuit/slimefun4/core/services/profiler/inspectors/ConsolePerformanceInspector.java

```diff
@@ -28,35 +30,61 @@ public class ConsolePerformanceInspector implements PerformanceInspector {
      */
     private final boolean verbose;
 
+    /**
+     * The order type of the timings.
+     */
+    private final SummaryOrderType orderType;
+
     /**
      * This creates a new {@link ConsolePerformanceInspector} for the given {@link CommandSender}.
      * 
      * @param console
-     *            The {@link CommandSender}, preferabbly a {@link ConsoleCommandSender}
+     *            The {@link CommandSender}, preferably a {@link ConsoleCommandSender}
      * @param verbose
      *            Whether the summary will be verbose or not
+     * @param orderType
+     *            The order type of the timings
      */
-    public ConsolePerformanceInspector(@Nonnull CommandSender console, boolean verbose) {
+    @ParametersAreNonnullByDefault
+    public ConsolePerformanceInspector(CommandSender console, boolean verbose, SummaryOrderType orderType) {
```

Comment:
This is also a breaking change if we can add a constructor that has a default orderType, can just be the normal when not actually adding any flag

Reply:
This should never be used outside of Slimefun and not seeing it used either - https://github.com/search?q=ConsolePerformanceInspector&type=code